post-type/module-base.php

```diff
@@ -167,6 +217,26 @@ public function p2p_init(){
                 'to'             => 'groups',
             ]
         );
+        /**
+         * Connection to Leaders
+         */
+        p2p_register_connection_type(
```

Comment:
@incraigulous this part shouldn't be needed

Reply:
@incraigulous I though you had added this piece. I see it was already there. Sorry about that.